config/application.rb

```diff
@@ -11,7 +11,11 @@
 module Publishers
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 7.0
+    config.load_defaults 7.1
+
+    config.pub_secrets = config_for(:secrets) # this line loads the config/secrets.yml file and store it in this namespace
+    # Raise error when a before_action's only/except options reference missing actions
+    config.action_controller.raise_on_missing_callback_actions = false
```

Comment:
should we fix the missing callbacks instead?

Reply:
Opened an issue to address in future PR https://github.com/brave-intl/creators-private-issues/issues/1664